### PR TITLE
Design Sorting/Filtering widgets for Sort Functionality

### PR DESCRIPTION
**Pull request by oluosiname**
Why?
Given that I am a user, I should be able to select options to sort product listings on pages that show multiple products.

How?
Designing a div to hold different sort and filter options

https://www.pivotaltracker.com/story/show/113281059

[finishes #113281059]

### CODE REVIEW

**Comment by makinwab** [comment]
@andela-oosiname why can't we view products in different categories? I also cannot see the widgets.

**Comment by makinwab** [comment]
Nice job @andela-oosiname. Let's reduce the height of the widget a bit. Also why don't we have the widget on the **categories** page i.e. when one clicks the **categories** link?

**Comment by oluosiname** [comment]
Hi @andela-bmakinwa. I've made the corrections and also added the widget to the categories page

**Comment by ydaniju** [comment]
Nice one there @andela-oosiname.

**Comment by ChigboIO** [comment]
This is cool :+1:

File: app/assets/javascripts/filter_widget.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$(document).ready(function(){
+  function setMinimum() {
+    var minVal = Number($("#lv").val()) + 1;
```
Comment (ChigboIO) [inline app/assets/javascripts/filter_widget.js]: I think we should give the `#lv` and `#hv` id a more readable name

Comment (oluosiname) [inline app/assets/javascripts/filter_widget.js]: Nicely caught @andela-echigbo, that's been taken care of.

**Comment by 0duaht** [comment]
Nice job @andela-oosiname. :+1:

**Comment by makinwab** [comment]
Can we have the `Apply` button's default style displayed and a little padding around it.  We should remove the `webkit-appearance` style from it.

**Pull request merged**